ChatQnA/benchmark_results.md

```diff
@@ -0,0 +1,161 @@
+  version: 1.3.0
+  modelUseHostPath: /home/sdp/opea_benchmark/model
+  HUGGINGFACEHUB_API_TOKEN: xxx
+  node: [1]
+  namespace: default
+  timeout: 1000 # timeout in seconds for services to be ready, default 30 minutes
+  interval: 5 # interval in seconds between service ready checks, default 5 seconds
+
+  services:
+    backend:
+      resources:
+        enabled: False
+        cores_per_instance: "16"
+        memory_capacity: "8000Mi"
+      replicaCount: [1, 2, 4, 8]
+
+    teirerank:
+      enabled: False
+      model_id: ""
+      resources:
+        enabled: False
+        cards_per_instance: 1
+      replicaCount: [1, 1, 1, 1]
+
+    tei:
+      model_id: ""
+      resources:
+        enabled: False
+        cores_per_instance: "80"
+        memory_capacity: "20000Mi"
+      replicaCount: [1, 2, 4, 8]
+
+    llm:
+      engine: vllm
+      model_id: "meta-llama/Llama-3.1-8B-Instruct" # mandatory
+      replicaCount:
+        with_teirerank: [7, 15, 31, 63] # When teirerank.enabled is True
+        without_teirerank: [8, 16, 32, 64] # When teirerank.enabled is False
+      resources:
+        enabled: False
+        cards_per_instance: 1
+      model_params:
+        vllm: # VLLM specific parameters
+          batch_params:
+            enabled: True
+            max_num_seqs: [256]
+          token_params:
+            enabled: False
+            max_input_length: ""
+            max_total_tokens: ""
+            max_batch_total_tokens: ""
+            max_batch_prefill_tokens: ""
+        tgi: # TGI specific parameters
+          batch_params:
+            enabled: True
+            max_batch_size: [1, 2, 4, 8] # Each value triggers an LLM service upgrade
+          token_params:
+            enabled: False
+            max_input_length: "1280"
+            max_total_tokens: "2048"
+            max_batch_total_tokens: "65536"
+            max_batch_prefill_tokens: "4096"
+
+    data-prep:
+      resources:
+        enabled: False
+        cores_per_instance: ""
+        memory_capacity: ""
+      replicaCount: [1, 1, 1, 1]
+
+    retriever-usvc:
+      resources:
+        enabled: False
+        cores_per_instance: "8"
+        memory_capacity: "8000Mi"
+      replicaCount: [1, 2, 4, 8]
+
+    redis-vector-db:
+      resources:
+        enabled: False
+        cores_per_instance: ""
+        memory_capacity: ""
+      replicaCount: [1, 1, 1, 1]
+
+    chatqna-ui:
+      replicaCount: [1, 1, 1, 1]
+
+    nginx:
+      replicaCount: [1, 1, 1, 1]
+
+benchmark:
+  # http request behavior related fields
+  user_queries: [4, 8, 16, 32, 64, 128, 256, 512, 1024]
+  concurrency: [1, 2, 4, 8, 16, 32, 64, 128, 256]
+  load_shape_type: "constant" # "constant" or "poisson"
+  poisson_arrival_rate: 1.0 # only used when load_shape_type is "poisson"
+  warmup_iterations: 10
+  seed: 1024
+
+  # workload, all of the test cases will run for benchmark
+  bench_target: [chatqna_qlist_pubmed]
+  dataset: ["/home/sdp/opea_benchmark/pubmed_10.txt"]
```

Comment:
this absolute path shouldn't be used here. and do you provide a step-by-step or a script to allow user to reproduce this data? pubmed_10.txt should be not a public dataset, right? need tell user how to get it.

Reply:
This yaml here want to show the really config used to generate this results, so I prefer to keep as it is.